Source/Fuse.Nodes/Visual.uno

```diff
@@ -95,7 +95,7 @@ namespace Fuse
 			UpdateIsContextEnabledCache();
 			UpdateIsVisibleCache();
 			UpdateContextSnapToPixelsCache();
-			WTIRooted();
```

Comment:
Why is this call removed? Is it called elsewhere?

Reply:
I think was part of a revert on duckers branch. I'll double check why it was done. There is clearly still a `WTIRooted` function and it's not called anywhere else.  Maybe just a merge gone bad somewhere.

I'll add a test that either proves we don't need this or fails.

Reply:
Odd, I can't figure out which commit actually makes that change (I viewed each commit on its own, each the merge from master).

Reply:
I've fixed this and added tests to make sure it'd fail if they go missing again.